.github/workflows/node.js.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         uses: actions/checkout@v2
 
       # Add keys to the SSH agent
-      - uses: webfactory/ssh-agent@v0.5.3 🖥️       
```

Comment:
This was main reason of failing deploy on master branch.
Icons should follow name, not expect location of job script